requirements.txt

```diff
@@ -1,3 +0,0 @@
-multi-key-dict==2.0.3
-pyowm==2.6.1
-requests>=2.13.0
```

Comment:
Looks like we still use the `HTTPError` module from requests

Reply:
Ok.  I have a question about that.  Many skills use the `IntentBuilder` class in Adapt, but adapt is not in their requirements.  My guess is that this is due to an assumption that adapt will already be in the virtual environment because core uses it.  The same could be said for the requests package in this case.  So, what is the best practice?  Explicitly include required packages that are already installed via core or omit packages we know are included in core?

Reply:
my 2 cents, i think its good practice to list any requirement you directly depend on

in the case of adapt, skills dont import it, they just use mycroft and mycroft itself depends on adapt, its a core technology so i would say we dont list that

a good rule of thumb is if you import it, list it in requirements, just be careful with version pinning (relax versions, let core decide when a package is common, better to break a skill then core)

Reply:
Yeah I like Jarbas's reasoning here, can we leave requests with no specified version?

Reply:
Added the requests package back in.